Formula/libomp.rb

```diff
@@ -3,6 +3,7 @@ class Libomp < Formula
   homepage "https://openmp.llvm.org/"
   url "https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/openmp-10.0.0.src.tar.xz"
   sha256 "3b9ff29a45d0509a1e9667a0feb43538ef402ea8cfc7df3758a01f20df08adfa"
+  license "MIT"
```

Comment:
Can you show me where did you see `MIT` license reference?

Reply:
happened to dig around with the llvm project's license recently.

Reply:
There's this line on the [homepage](https://openmp.llvm.org):

> All of the code here is dual licensed under the MIT license and the UIUC License (a BSD-like license). The LICENSE.txt file at the top of the OpenMP project contains the license text and associated patent grants.

Reply:
@Rylan12 is correct -- that is precisely where I got it from.

Reply:
Sounds good! 👍